packages/dds/tree/src/schema-stored/comparison.ts

```diff
@@ -130,14 +131,26 @@ export function allowsFieldSuperset(
     if (original.kind === FieldKind.Forbidden) {
         return true;
     }
-    if (superset.types === undefined) {
+    return allowsTypesSuperset(original.types, superset.types);
+}
+
+/**
+ * @returns true iff `superset` is a superset of `original`.
+ *
+ * This does not require a strict (aka proper) superset: equivalent schema will return true.
+ */
+export function allowsTypesSuperset(
```

Comment:
nit: This name is a bit hard to read for me. Is it supposed to be `allowedTypesSuperset`?

Reply:
Oh, I see the same scheme is used for other things. What's the meaning behind it?

Reply:
All these check is one schema allows a non-strict superset of what another schema allows, for various different types of schema.

Name for this one is a bit strange/abstract since we can't name it allowsTreeSuperset, since thats taken, and this only really means that for Value fields.

Reply:
Ahh I see, what I was missing is that these are meant to be used with schema validation. I was mostly confused on the 'allows' part of the name cause it seemed like something like `isTypesSuperset` more accurately describes the function.

Reply:
I renamed allowsTypesSuperset to allowsTreeSchemaIdentifierSuperset. I think thats better.